src/components/News/SingleNews.js

```diff
@@ -9,41 +9,111 @@ import { Translate, English, Swedish } from '../Translate'
+      <h2>
+        <Translate>
+          <English>Event Details</English>
+          <Swedish>Eventinformation</Swedish>
+        </Translate>
+      </h2>
+      <p>
+        <b>
+          <Translate>
+            <English>Location</English>
+            <Swedish>Plats</Swedish>
+          </Translate>
+        </b>
+        <br />
+        {item.eventLocation}
+      </p>
+      <p>
+        <b>
+          <Translate>
+            <English>Start Time</English>
+            <Swedish>Starttid</Swedish>
+          </Translate>
+        </b>
+        <br />
+        {new Date(item.eventStartTime)
+            .toLocaleDateString(
+              lang === 'en' ? 'en-US' : 'sv-SE',
+              {
+                day: 'numeric',
+                month: 'long',
+                year: 'numeric',
+                hour: 'numeric',
+                minute: 'numeric'
+              }
+            )}
+      </p>
+      <p>
+        <b>
+          <Translate>
+            <English>End Time</English>
+            <Swedish>Sluttid</Swedish>
+          </Translate>
+        </b>
+        <br />
+        {new Date(item.eventEndTime)
+            .toLocaleDateString(
+              lang === 'en' ? 'en-US' : 'sv-SE',
+              {
+                day: 'numeric',
+                month: 'long',
+                year: 'numeric',
+                hour: 'numeric',
+                minute: 'numeric'
+              }
+            )}
+      </p>
+    </div>
+  </div>
+
+
 export const SingleItem = ({ item, location, lang, match }) =>
 <Calypso type='item' search={'/' + match.params.postId}>
-  {(item) =>
+  {(item) => // item CAN be undefined here
     <Fragment>
       <Title>
-          <Translate>
-            <English>{ `${item.titleEnglish} - Konglig Datasektionen` }</English>
-            <Swedish>{ `${item.titleSwedish} - Konglig Datasektionen` }</Swedish>
-          </Translate>
-        </Title>
-        <header key="header">
-          <div className="header-inner">
-            <div className="row">
-              <div className="header-left col-md-2">
-                <Translate>
-                  <English>
-                    <Link to="/en/news">{'« '}Back</Link>
-                  </English>
-                  <Swedish>
-                    <Link to="/nyheter">{'« '}Tillbaka</Link>
-                  </Swedish>
-                </Translate>
-              </div>
-              <div className="col-md-8">
-                <h2>
-                  <Translate>
-                    <English>News</English>
-                    <Swedish>Nyheter</Swedish>
-                  </Translate>
-                </h2>
-              </div>
-              <div className="header-right col-md-2"/>
-            </div>
-          </div>
-        </header>
+        <Translate>
+          <English>{`${item ? item.titleEnglish : "????"} - Konglig Datasektionen`}</English>
+          <Swedish>{`${item ? item.titleSwedish : "????"} - Konglig Datasektionen`}</Swedish>
```

Comment:
????

Reply:
!!!!!

Reply:
(it is an edgecase, and only appears in the browser titlebar, so I thought it was kind funny to keep it that way)